test/test.rs

```diff
@@ -29,7 +29,6 @@ mod test_poll;
     target_os = "haiku"
 )))]
 mod test_pty;
-mod test_resource;
```

Comment:
`resource.rs` and `timer.rs` are under `src/sys`, so I moved `test_resource.rs` and `test_timer.rs` from `test/` to `test/sys/`